backend/lib/plugins/auth.js

```diff
@@ -22,10 +23,42 @@ const checkAuth = async (req) => {
   req.userId = mongoose.Types.ObjectId(user[auth.jwtMongoIdKey]);
 };
 
+const cookieMaxAge = 60 * 60 * 24 * 7; // 1 week; TODO: ask about alternate?
+const tokenCookieOptions = () => {
+  let domain;
+  switch (env) {
```

Comment:
`AUTH_APP_DOMAIN` or actually better `APP_DOMAIN` would be okay.

Leveraging `AUTH_APP_URL` is a little awkward as it's really a dummy value for our Auth0 scoped parameter. We'd need to use some ugly regex match as below and that's just super hard to reason about and potentially error prone.

I think this will work unless we change the AUTH_APP_URL to something totally unrelated and random. Likely not the right approach.

```
domain: env === "dev" ? "localhost" : AUTH_APP_URL.match(/https?:\/\/(\w*\.){1,}(?<domain>\w*\.\w*)\/$/).groups.domain
```

Reply:
This is done I set as `APP_DOMAIN` with `localhost` as default so all the devs should be good without needing to update anything. The other envs have it set in `main.tf`.